backend/core/Helpers/CsvHelper.cs

```diff
@@ -43,10 +42,14 @@ public static string ConvertToCSV<T>(this IEnumerable<T> data, string delimiter
         /// </summary>
         /// <typeparam name="T"></typeparam>
         /// <param name="data"></param>
+        /// <param name="tableName"></param>
         /// <returns></returns>
-        public static DataTable ConvertToDataTable<T>(this IEnumerable<T> data)
+        public static DataTable ConvertToDataTable<T>(this IEnumerable<T> data, string tableName)
         {
-            var dt = new DataTable();
+            var dt = new DataTable()
+            {
+                TableName = tableName
```

Comment:
I was hoping the table name would be used in the Excel report, but it isn't for some reason.